infrastructure/preview.tfvars

```diff
@@ -3,3 +3,4 @@ idam_api_url = "https://idam-api.aat.platform.hmcts.net"
 document_management_valid_domain = "^https?://(?:api-gateway\\.preprod\\.dm\\.reform\\.hmcts\\.net|dm-store-aat\\.service\\.core-compute-aat\\.internal(?::\\d+)?)"
 default_print_url = "https://return-case-doc-ccd.nonprod.platform.hmcts.net/jurisdictions/:jid/case-types/:ctid/cases/:cid"
 frontend_url = "https://www-ccd.aat.platform.hmcts.net"
+elastic_search_timeout=8000
```

Comment:
we can remove this one, we have the default value at 6 sec which is good for preview

Reply:
perfect :)